src/Dev/Deprecation.php

```diff
@@ -77,6 +77,11 @@ class Deprecation
      */
     private static bool $showNoReplacementNotices = false;
 
+    /**
+     * @internal
+     */
+    private static bool $showCalledFromSupportedCodeNotices = false;
```

Comment:
```suggestion
    private static bool $showNoticesCalledFromSupportedCode = false;
```
Makes it clearer we're showing notices which have been called from supported code. Otherwise we're showing something that has to do with "code notices".

Reply:
Please update the accessor methods as well.

Reply:
Done